static/js/src/advantage/countries-and-states.js

```diff
@@ -0,0 +1,368 @@
+export const vatCountries = [
```

Comment:
Can these values be returned from the server instead of loaded into the bundle?

Reply:
@albertkol ? :smiley: